spec/models/participant_profile/ecf_spec.rb

```diff
@@ -334,4 +334,47 @@
       expect(subject.record_to_serialize_for(lead_provider:)).to eq(subject.induction_records.latest.reload)
     end
   end
+
+  describe "#relevant_induction_record_for" do
+    let(:cpd_lead_provider) { create(:cpd_lead_provider, :with_lead_provider) }
+    let(:lead_provider) { cpd_lead_provider.lead_provider }
+    let(:cohort) { Cohort.next || create(:cohort, :next) }
```

Comment:
I think we can use current as now it's set to 2022?

Reply:
Hi @cwrw Addressed in https://github.com/DFE-Digital/early-careers-framework/pull/2920/commits/653fbb7cf679581af150c659d0e675c0e7f27203 tks 👍